src/ui.button/configs/default.config.js

```diff
@@ -121,7 +94,7 @@ export default /*tw*/ {
         color: "white",
         variant: "secondary",
         class: `
-          border-white bg-transparent text-white
+          border-white bg-transparent text-gray-500
```

Comment:
bg-transparen >>> bg-white
text-gray-500 >>> text-gray-900

hover / focus = text-gray-800
active = text-gray-700

Reply:
`hover: text-gray-800` - need to be fixed and apply please those changes for all 3 variants of white button.